content/security/for-admins/enforce-sign-in/methods.md

```diff
@@ -0,0 +1,302 @@
+5. Configure the new registry item to match the registry script you created, specifying the action as **Update**. Make suer you input the correct path, value name (`allowedOrgs`), and value data (your organization’s name).
+6. Link the GPO to an Organizational Unit (OU) that contains the machines you want to apply this setting to.
+7. Test the GPO. Test the GPO on a small set of machines first to ensure it behaves as expected. You can use the `gpupdate /force` command on a test machine to manually refresh its group policy settings and check the registry to confirm the changes.
+8. Once verified, you can proceed with broader deployment. Monitor the deployment to ensure the settings are applied correctly across the organization's computers.
+
+## plist method (Mac only)
+
+1. Create the file `/Library/Application Support/com.docker.docker/desktop.plist`
+2. Open `desktop.plist` in a text editor and add the following content, where `myorg` is replaced with your organization’s name all lowercase:
+
+   ```xml
+   <?xml version="1.0" encoding="UTF-8"?>
+   <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+   <plist version="1.0">
+     <dict>
+	     <key>allowedOrgs</key>
+	     <array>
+             <string>myorg</string>
+         </array>
+     </dict>
+   </plist>
+   ```
+   > **Important**
+   >
+   > Only one entry for `allowedOrgs` is currently supported. If you add more than one value, sign-in enforcement silently fails.
+   { .important }
+
+3. Modify the file permissions to ensure the file cannot be edited by any non-administrator users.
+4. Restart Docker Desktop. 
+5. Open Docker Desktop and when Docker Desktop starts, verify that the **Sign in required!** prompt appears.
+
+### Example deployment 
+
+The following is only an illustrative example. 
+
+There are many ways to deploy the `.plist` file. The method you choose is dependent on your organizations infrastructure, security policies, and the administrative rights of the end-users. 
+
+{{< tabs >}}
+{{< tab name="MDM" >}}
+
+1. Follow the steps outlined above to create the `desktop.plist` file.
+2. Use an MDM tool like Jamf or Fleet to distribute the `desktop.plist` file to `/Library/Application Support/com.docker.docker/` on target macOS devices.
+3. Through the MDM tool, set the file permissions to allow editing by administrators only.
+
+{{< /tab >}}
+{{< tab name="Shell script" >}}
+
+1. Create a Bash script that can check for the existence of the `.plist` file in the correct directory, create or modify it as needed, and set the appropriate permissions.
+   Include commands in your script to:
+    - Navigate to the `/Library/Application Support/com.docker.docker/` directory or create it if it doesn't exist.
+    - Use the `defaults` command to write the required keys and values to the `desktop.plist` file. For example, `defaults write /Library/Application\ Support/com.docker.docker/desktop.plist allowedOrgs -string "myorg"`.
```

Comment:
The example `defaults write...` is a little long when I view the deploy preview and ends with the period on a new line. Is it work using a code block here instead?